docs/machine-learning/how-to-guides/train-machine-learning-model-cross-validation-ml-net.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET"
+description: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+ms.date: 04/17/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use cross validation to train a machine learning model in ML.NET
+---
+
+# How-To: Learn how to train and evaluate a machine learning model using cross validation in ML.NET
+
+Learn how to use cross validation to build more robust machine learning models in ML.NET. Although this sample uses a regression model, it is applicable to other algorithms in ML.NET.
```

Comment:
>earn how to use cross validation [](start = 1, length = 32)

do we need to say what cv is, or link to something explaining CV?

Reply:
#resolved